src/FSharp.Analyzers.SDK.Testing/FSharp.Analyzers.SDK.Testing.fsi

```diff
@@ -2,13 +2,16 @@ module FSharp.Analyzers.SDK.Testing
 
 open System.Threading.Tasks
 open FSharp.Compiler.CodeAnalysis
+open FSharp.Compiler.Diagnostics
 
 type FSharpProjectOptions with
 
     static member zero: FSharpProjectOptions
 
 type Package = { Name: string; Version: string }
 
+exception DiagnosticErrors of FSharpDiagnostic array
```

Comment:
Should we suffix this with `Exception`, feels like the dotnet thing to do here?

Reply:
Hmm, nah that doesn't seem to be a convention in the fsharp compiler codebase.
We could maybe go with `CompilerDiagnosticErrors` but I'm not hellbent on that either.